src/client/base/browser_clock.ts

```diff
@@ -0,0 +1,31 @@
+import { Clock } from '../../shared/base/clock'
+
+export type CancelTimer = () => void
+
+function setTimeout(cb: (...args: any[]) => void, seconds: number): CancelTimer {
+  const handle = window.setTimeout(cb, seconds * 1e3)
+  return window.clearTimeout.bind(null, handle)
+}
+
+function setInterval(cb: (...args: any[]) => void, seconds: number): CancelTimer {
+  const handle = window.setInterval(cb, seconds * 1e3)
+  return window.clearInterval.bind(null, handle)
+}
+
+function setImmediate(cb: (...args: any[]) => void): CancelTimer {
+  const handle = window.setTimeout(cb, 0)
+  return window.clearTimeout.bind(null, handle)
+}
+
+function performanceNow() {
+  return window.performance.now() * 1e-3
+}
+
+
+export const BrowserSystemClock: Clock = {
```

Comment:
Hm I was planning to use this for fading them out over time, but I excluded that from this PR. So this had nothing to do with the added changes but we will surely be needing this one anyway.